assets/js/blocks/product-search/index.js

```diff
@@ -120,6 +122,8 @@ registerBlockType( 'woocommerce/product-search', {
 
 	/**
 	 * Save the props to post content.
+	 *
+	 * @param {Array} attributes Props to pass to block.
```

Comment:
Should be `{Object}` (should be updated for all other instances of new docs for the `save` property).